diff --git a/.gitmodules b/.gitmodules
new file mode 100644
index 0000000..9699fe5
--- /dev/null
+++ ./.gitmodules
@@ -0,0 +1,3 @@
+[submodule "recoll-webui"]
+	path = recoll-webui
+	url = https://github.com/koniu/recoll-webui.git
diff --git a/Dockerfile b/Dockerfile
index 92ac2b1..9efddf7 100644
--- ./Dockerfile
+++ ./Dockerfile
@@ -1,6 +1,8 @@
 FROM debian:jessie
 MAINTAINER Jivan Kulkarni <jivank@gmail.com>
 
+RUN apt-get update && apt-get install -y --force-yes apt-transport-https ca-certificates
+
 RUN echo deb http://www.lesbonscomptes.com/recoll/debian/ unstable main > \
 	/etc/apt/sources.list.d/recoll.list
 
diff --git a/docker-compose.yml b/docker-compose.yml
new file mode 100644
index 0000000..cb0bb7d
--- /dev/null
+++ ./docker-compose.yml
@@ -0,0 +1,12 @@
+version: '2.1'
+
+services:
+    recoll-webui:
+        build:
+            context: .
+            dockerfile: Dockerfile
+        image: huangyingw/docker-recoll-webui-pdfocr
+        volumes:
+            - ~/myproject/git/AI/coursera/deeplearning.ai/:/data/
+        ports:
+            - '8080:8080'
diff --git a/recoll-webui b/recoll-webui
new file mode 160000
index 0000000..c3151ab
--- /dev/null
+++ ./recoll-webui
@@ -0,0 +1 @@
+Subproject commit c3151abc2b4416fb4a63b7833b8c99d17e3ace6e
